lib/suspenders/app_builder.rb

```diff
@@ -275,6 +275,10 @@ def setup_stylesheets
         'app/assets/stylesheets/application.css.scss'
     end
 
+    def install_refills
+      run "rails generate refills:import flashes --only-stylesheets"
```

Comment:
Eric added `--only-stylesheets` but that's not an option to refills.

I think it does make sense for refills to generate markup and CSS, as they are coupled by definition, but we generate our own version of the markup in suspenders. @tabfugnic would you provide a step here to remove the refills-generated flashes partial?

Thank you!